### PR TITLE
Fixes on labels

### PR DESCRIPTION
**Pull request by zarov**
A few fixes on labels:

- **fix(label): makes LabelLayer able to run alone**
Before, no configuration could be passed to it, like a source. This
fixes it.

- **feat(label): add a new class to Labels**
The class corresponds to `itowns-label-[layer-id]`. Documentation has
been added as well.

- **fix(label): allow label to only have an icon**

- **fix(label): labels don't jitter anymore**
There was a rounding that was done when comparing positions in the
renderer. This is still used for comparison, but the assigned value is
not the rounded one anymore.

### CODE REVIEW

File: src/Core/View.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ function _preprocessLayer(view, layer, parentLayer) {
     if (layer.isLabelLayer) {
         view.mainLoop.gfxEngine.label2dRenderer.registerLayer(layer);
     } else if (layer.labelEnabled) {
-        const labelLayer = new LabelLayer(`${layer.id}-label`, view.referenceCrs);
+        const labelLayer = new LabelLayer(`${layer.id}-label`, { crs: view.referenceCrs });
```
Comment (gchoqueux) [inline src/Core/View.js]: `crs` will be always `view.referenceCrs`?

File: src/Layer/LabelLayer.js [hunk 2]
```diff
@@ -86,7 +101,13 @@ class LabelLayer extends Layer {
 
                 const geometryField = g.properties.style && g.properties.style.text.field;
                 if (!geometryField && !featureField && !layerField) {
-                    return;
+                    // Check if there is an icon, with no text
```
Comment (gchoqueux) [inline src/Layer/LabelLayer.js:102]: could you remove the checking in `Label`?

Comment (zarov) [reply src/Layer/LabelLayer.js:102]: Done

File: src/Core/View.js (hunk 1 above)
Comment (zarov) [reply src/Core/View.js]: Yes, I change things in `LabelLayer` and removed that

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**